website/source/docs/providers/aws/r/kms_alias.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "aws"
+page_title: "AWS: aws_kms_alias"
+sidebar_current: "docs-aws-resource-kms-alias"
+description: |-
+  Provides a display name for a customer master key.
+---
+
+# aws\_kms\_alias
+
+Provides a KMS customer master key. AWS Console enforces 1-to-1 mapping between aliases & keys,
```

Comment:
fixed & squashed into the original commit with docs, so I'm not cluttering the git log even more.